WL-3743 Updated search partiticipants in site_info as per new test plan.

ouit0408 opened the pull request:
Have added new variable to STATE_SITE_PARTICIPANT_LIST in SiteAction class
to save the original site members list. Search is now case insensitive.

buckett commented:
You don't have to change this future, a comment like:

```
WL-3743 When filtering participants, actions still work.

Have added new variable to STATE_SITE_PARTICIPANT_LIST in SiteAction class
to save the original site members list. Search is now case insensitive.
```

Makes it easier to see what the change is doing without having to go to the Jira.

buckett commented:
I think you also want to remove the attribute from the session in the `cleanState(SessionState)` call in `SiteAction`.

ouit0408 committed:
WL-3743 When searching for users , all actions will work.

Have added new variable to STATE_SITE_PARTICIPANT_LIST in SiteAction class
to save the original site members list. Search is now case insensitive.

ouit0408 commented:
Made the change.
